content/contribute/_index.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
All this "how to work on Crossplane" stuff does feel like it probably belong in the c/c repo more than the docs repo. Maybe we move it back to a `contributing` or `contributing-docs` dir there and link from here?

Reply:
All of this has been removed and the contrib guide is only about contributing to the docs.